app/code/Magento/Quote/Test/Unit/Model/QuoteManagementTest.php

```diff
@@ -416,9 +429,12 @@ public function testCreateEmptyCartForCustomerReturnExistsQuote()
         $this->model->createEmptyCartForCustomer($userId);
     }
 
-    public function testAssignCustomerFromAnotherStore()
+    /**
+     * @return void
+     */
+    public function testAssignCustomerFromAnotherStore(): void
     {
-        $this->expectException('Magento\Framework\Exception\StateException');
```

Comment:
AFAIK It was intentionally done in order not use ::class due to performance reasons - on big data sets it's noticeable when php unit tries additionally lookup for exception class.
@lbajsarowicz, am I correct?